rust-toolchain-nightly.toml

```diff
@@ -0,0 +1,7 @@
+[toolchain]
+channel = "nightly-2023-07-31"
```

Comment:
better to sync nightly version with Parity 
https://github.com/paritytech/scripts/blob/f901682d2d1b2fa5ed046864f36c2da481532843/dockerfiles/ci-linux/Dockerfile#L7